daemon/src/db.rs

```diff
@@ -160,6 +173,14 @@ impl Connection {
         .bind(&cfd.settlement_time_interval_hours().whole_hours())
         .bind(&cfd.quantity())
         .bind(&cfd.counterparty_network_identity())
+        .bind(&cfd.counterparty_libp2p_identity().unwrap_or({
```

Comment:
❓ Could you insert the correct value here (using a similar `match` to the one you use below) if you detect a `Libp2pIdentity::placeholder()`?

Reply:
I would still need it when loading, because previously opened positions should also be treated with the new protocol on the taker. Given that I just did it on loading. I don't want to update existing CFDs.

Reply:
Note on:

> I could even do it in the migration statement using a CASE

No we can't. I gave this a try but it appears this is not possible in Sqlite.